campusmap/src/info/Info/Info.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { ReactElement } from 'react';
+import { Card, Col, Row, Carousel, Jumbotron, Button, Container, Form } from 'react-bootstrap';
+import { Location } from 'campusmap/src/types';
+
+
+interface Props {
+    location: Location;
+}
+
+/**
+ * Individual Info Result component rendered in a carousel.
+ */
+
+ const Info = ({ location }: Props): ReactElement => {
+    return (
+        <body>
+            <Jumbotron>
+                <Card className="mb-4">
+                    <Row className="p-4">
+                        <Col className="col-2">
+                            <Card.Img src={"..."} style={{ maxWidth: '100px' }} alt={location.properties.name} />
+                        </Col>
+                        <Col className="col-10">
+                            <Card.Body>
+                            <Card.Title>{location.properties.name}</Card.Title>
+                            <Card.Text>{location.properties.description}</Card.Text>
+                            {location.properties.nick 
+                                && <Card.Subtitle>Also known as:&nbsp;{location.properties.nick}</Card.Subtitle>}
+                            </Card.Body>
+                        </Col>
+                        <Button variant="outline-danger" href="`/info?loc=${location._id}`">Show on Map</Button>
+                    </Row>
+                </Card>
+                <Carousel>
+                    <Carousel.Item>
+                        <img 
+                            src="http://www.alpha6294.com/History/ThetaXionCampus/DarrinCommunicationsCenter/Building/DarrinCommunicationsCenter.jpg"
+                            className="d-block w-100" 
+                            alt="..."
+                        />
+                    </Carousel.Item>
+                    <Carousel.Item>
+                        <img 
+                            src="https://www.bing.com/th?id=OIP.npiBC_gFZvex2KDN16YTZAHaFj&pid=Api&rs=1&p=0"
+                            className="d-block w-100"                   
+                            alt="..."
+                        />
+                    </Carousel.Item>
+                </Carousel>
+            </Jumbotron>
```

Comment:
This can be its own separate component, `InfoPagePhotoCarousel` or something similar. This file is more similar to a top level `InfoPage`